### ISSUE TITLE
Different templates per folder

### ISSUE THREAD

**Issue by larsrinn** [LostPaul/obsidian-folder-notes#89]
Hi @LostPaul - I just started using your plugin and it seems really helpful. One thing I've stumbled across is: Is it possible to define the template on a per (top-level)-folder-level?

**Comment by LostPaul** [comment]
> Hi @LostPaul - I just started using your plugin and it seems really helpful. One thing I've stumbled across is: Is it possible to define the template on a per (top-level)-folder-level?

No, it's not possible yet but thank you for the suggestion I'll consider adding it within the next releases.

**Comment by larsrinn** [comment]
Thanks, I just bought you a coffee to accelerate it's arrival ;)

**Comment by LostPaul** [comment]
Thank you! I might do it on the next weekend :)
